converter/webtool_pdf2csv.py

```
@@ -0,0 +1,133 @@
+                        new_row["経度"] = None
+                        new_row["説明"] = pos_dir.get("説明", "") + " (位置情報が見つかりませんでした)"
+                    df = pd.concat([df, pd.DataFrame([new_row])], ignore_index=True)
+    os.remove(pdf_path)  
+    placeholder1.text("処理が完了しました。")
+    content_placeholder.write(df)
+    csv_filename = f"{base_name}.csv"
+    csv_path = os.path.join(tmp_dir, f"{csv_filename}")
+    df.to_csv(csv_path, index=False)  
+    st.download_button(
+        'ダウンロード',
+        open(csv_path, 'rb'),
```

Ruff (0.11.9)

Use a context manager for opening files

(SIM115)